### ISSUE TITLE
'nearbait_size' for Different Model Organisms

### ISSUE THREAD

**Issue by MediciPrime** [rr1859/R.4Cker#18]
@rr1859 what is the reasoning behind setting the 'nearbait_size' parameter at 1e6 for 4bp RE and 5e6 for 6bp RE?  Also how does the 'frag_window' parameter fit in?  

I am asking because I realize that the sample data was generated using M. musculus while my data is from D. melanogaster.  

Will the varying chrom-sizes for the different model organisms impact the results in anyway?  

If so then should I change the value for 'nearbait_size' to better fit my model organism and what value would you recommend?

**Comment by rr1859** [comment]
Hi, 

We used those sizes based on the decrease in signal around the bait. As per this paper (http://www.sciencedirect.com/science/article/pii/B9780123919380000045) they use 2 MB for 6bp cutters and 200kb for 4bp but we extended it to better capture the decrease in signal in our model. 

From what has been reported for Drosophila, the TADs are smaller than in mammalian genomes so you are right it might be more appropriate to use a smaller window. That being said I havnt analyzed a lot of data from Drosophila to determine what the size should be. For now I would recommend trying values ranging from 2e5 to 1e6. 

Hope this helps.
